Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -950,7 +950,7 @@ def setup_(self, step, stepName, stepDict, k, properties):
             dd4hepGeom+=stepDict[step][k]['--geometry']
             stepDict[stepName][k] = merge([{'--geometry' : dd4hepGeom, '--procModifiers': 'dd4hep'}, stepDict[step][k]])
     def condition(self, fragment, stepList, key, hasHarvest):
-        return ((fragment=='TTbar_13' or fragment=='ZMM_13' or fragment=='SingleMuPt10') and '2021' in key) \
+        return ((fragment=='TTbar_13' or fragment=='TTbar_14' or fragment=='ZMM_13' or fragment=='SingleMuPt10') and '2021' in key) \
```

Comment:
since we won't use dd4hep for run2, perhaps 13 tev workflows aren't very useful?

Reply:
Since we have only 11624.911 in pre2, I think it is good to have both 11624.911 and 11634.911 in pre3

Reply:
I also agreed with @davidlange6 that 13TeV isn't useful to keep